scss/modules/_helpers.scss

```diff
@@ -322,8 +322,7 @@
   @media only screen and (min-width : $breakpoint-medium) {
     .priority-0,
     .not-for-small {
-      position: relative;
-      left: auto;
+      display: block;
```

Comment:
I think a better method would be to wrap the display: none in a max-width media query. This will set all items to display: block if it has priority-0. This is not always desired.